docs/requirements.txt

```diff
@@ -72,7 +72,7 @@ snowballstemmer==2.2.0
     # via sphinx
 soupsieve==2.4.1
     # via beautifulsoup4
-sphinx==7.2.2
+sphinx==7.2.6
```

Comment:
This should've been bumped separately. There's no evidence of Sphinx breaking in CI/RTD: https://app.readthedocs.org/projects/pip-tools/builds/. And it's unrelated.